### DISCUSSION TITLE
Handling of new attribut wastewater_node.wwtp_number

### DISCUSSION THREAD

**Discussion by urskaufmann** [teksi/wastewater#32] [Ideas]
How do we handle this new attribut? Every node should get this wwtp_number (ARA_Nummer). Also if not connected, just if the geometry is in the area of the wwtp.
Till now, there was no table wwtp in QGEP. I can add the wastewater_treatmant_plant-layer to the project and define a value_relation. And the user has to select.
Or TEKSI develops a new function, that generate the number out of the geometry and the user has not to care for it.
Or we ignore this attribut in the moment..

**Comment by cymed** [comment]
How about setting it via a QGIS project variable? 
(also a good solution for fk_provider and fk_dataowner, imho, as they rarely change. Maybe even fk_owner and fk_operator based on the function hierarchic)
